subprojects/gradle/module-dependencies-graph/src/main/kotlin/com/avito/module/internal/dependencies/FindAndroidAppTaskAdvisor.kt

```diff
@@ -0,0 +1,49 @@
+package com.avito.module.internal.dependencies
+
+import com.avito.module.internal.dependencies.FindAndroidAppTaskAction.Verdict.MultipleSuitableApps
+import com.avito.module.internal.dependencies.FindAndroidAppTaskAction.Verdict.NoSuitableApps
+import com.avito.module.internal.dependencies.FindAndroidAppTaskAction.Verdict.OneSuitableApp
+
+internal class FindAndroidAppTaskAdvisor {
+
+    fun giveAdvice(verdict: FindAndroidAppTaskAction.Verdict): String {
+        return when (verdict) {
+            is OneSuitableApp -> "In your project is only one suitable app ${verdict.project.first.path}"
+            is MultipleSuitableApps -> verdict.advice()
+            is NoSuitableApps -> "In your project are no suitable android apps"
```

Comment:
```suggestion
            is NoSuitableApps -> "There are no suitable Android apps"
```
Missed subjects in sentences. Can rephrase this way

Reply:
Done